dev.edfloreshz.CosmicTweaks.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "dev.edfloreshz.CosmicTweaks",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "cosmic-ext-tweaks",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:rw"
```

Comment:
You made the exception for ro but this is rw Which one is needed ?

Reply:
Oops, sorry I missed that, rw is the one needed.

Reply:
Here's the [PR](https://github.com/flathub-infra/flatpak-builder-lint/pull/487)